src/pages/Login/Login.tsx

```diff
@@ -36,19 +36,24 @@ export default function Login() {
           idToken: idToken
         })
       })
+
+
       if (login.ok) {
         // User exists so go to /dashboard
         localStorage.setItem('member_id', user.uid);
         localStorage.setItem('member_name', displayName || "guest");
```

Comment:
I don't think we need this anymore right? Since we should be using `auth.currentUser`

Reply:
yea we can remove this